docs/errors.md

```
@@ -0,0 +1,584 @@
+config.Verbose = false  // Collapsed mode
+config.Color = "auto"   // auto, always, never
+
+formatted := errUtils.Format(err, config)
+fmt.Fprint(os.Stderr, formatted)
+```
+
+### Structured Markdown Output
+
+Errors are formatted as structured markdown with hierarchical sections:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)